libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -1967,4 +1990,22 @@ bool PbftManager::checkBlockWeight(const PeriodData &period_data) const {
   return true;
 }
 
+blk_hash_t PbftManager::getLastPbftBlockHash() { return pbft_chain_->getLastPbftBlockHash(); }
+
+std::vector<std::shared_ptr<Vote>> PbftManager::getRewardVotes(const std::vector<vote_hash_t> &reward_votes_hashes) {
+  std::unordered_set<vote_hash_t> reward_votes_hashes_set;
+  for (const auto &v : reward_votes_hashes) reward_votes_hashes_set.insert(v);
+
+  auto reward_votes = vote_mgr_->getRewardVotes();
+  for (auto it = reward_votes.begin(); it != reward_votes.end();) {
+    if (reward_votes_hashes_set.contains((*it)->getHash())) {
+      ++it;
+    } else {
+      it = reward_votes.erase(it);
+    }
+  }
+
+  return reward_votes;
+}
```

Comment:
why this is not in vote_manager? it has nothing to do with pbft

Reply:
PbftManager::getRewardVotes only return reward votes that are list in PBFT block. The PbftManager::getRewardVotes is a private method in PBFT class.
The method name is not good. I change name to getRewardVotesInBlock